### DISCUSSION TITLE
Works with Apache?

### DISCUSSION THREAD

**Discussion by ghost** [modoboa/modoboa#2198] [Help] [answered]
I already have my VPS setup with a mail system, but this has more features and is just _better_. 

My main worry is with Apache. I already have 50+ Apache VHosts.

Does Modoboa require a fresh install - or will it work with my existing system?

**Comment by tonioo** [comment, marked as answer]
Yes, modoboa does work with Apache but you won't be able to use the installer. You must follow the manual procedure described here: https://modoboa.readthedocs.io/en/latest/installation.html#manual-installation
